src/components/FullPage.js

```diff
@@ -64,6 +65,10 @@ class FullPage extends React.Component {
   }
 
   scrollToSlide(slide) {
+    if(this.props.onChange !== undefined) {
```

Comment:
I think it is better to provide empty function to default props. I have a branch with the eslint recommended rules and there is a rule to explicitly set all non-required stuff in defaultProps. I also found this while researching https://twitter.com/rem/status/889801092388835329